line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingClient.java

```diff
@@ -68,6 +70,19 @@
      */
     CompletableFuture<BotApiResponse> pushMessage(PushMessage pushMessage);
 
+    /**
+     * Send messages to users when you want to.
+     *
+     * <p>INFO: Use of the Push Message API is limited to certain plans.
+     *
+     * @throws ConflictException Message with the retryKey was already sent.
+     *
+     * @see #replyMessage(ReplyMessage)
+     * @see <a href="https://developers.line.me/en/reference/messaging-api/#send-push-message">//developers.line.me/en/reference/messaging-api/#send-push-message</a>
+     */
+    CompletableFuture<BotApiResponse> pushMessage(UUID retryKey, PushMessage pushMessage)
```

Comment:
`retryKey` is optional parameter, how about putting it last?

Reply:
nn~ in my opinion, creating option bean is over-engineering.

and so retryKey is an optional parameter but the parameter is an HTTP header. Put the HTTP header as a first parameter is a natural way, maybe.

@imasahiro how do you think about this?

Reply:
I prefer to keep it as is. 
Rather than adding RequestOption, it might be better to provide an automatic retry feature at new interface (e.g. RetryableLineMessagingClient) on the SDK side for better usability?

Reply:
OK. You can keep it as is.

>  it might be better to provide an automatic retry feature at new interface (e.g. RetryableLineMessagingClient) on the SDK side for better usability?

That's good idea.